code/modules/mob/living/simple_animal/friendly/drone/inventory.dm

```diff
@@ -112,4 +112,31 @@
 
 
 /mob/living/simple_animal/drone/stripPanelEquip(obj/item/what, mob/who, where)
-	..(what, who, where, 1)
\ No newline at end of file
+	..(what, who, where, 1)
+
+/mob/living/simple_animal/drone/verb/quick_equip()
+	set name = "quick-equip"
+	set hidden = 1
+	if (isdrone(src))
```

Comment:
The typecasting thing is done in human inventory [here](https://github.com/tgstation/-tg-station/blob/master/code/modules/mob/living/carbon/human/inventory.dm#L162), which is why I thought there was a reason for it being done that way